panoptes/utils/images/fits.py

```diff
@@ -466,6 +466,14 @@ def update_observation_headers(file_path, info):
             which includes basic information about the observation.
     """
     with fits.open(file_path, 'update') as f:
+
+        # Let us check the headers
+        >>> fits_fn = getfixture('unsolved_fits_file')
+        >>> info = {'field_name': 'M42'}
+        >>> update_observation_headers{fits_fn, info}
+        >>> fits_utils.getval(fits_fn, 'FIELD')
+		M42
+
```

Comment:
This needs to go inside the docstring.

It also looks like the `M42` is indented too much.

Reply:
Thanks, will make the changes.